plugins/techdocs-backend/src/service/helpers.ts

```diff
@@ -106,22 +106,16 @@ export class DocsBuilder {
     const buildMetadataStorage = new BuildMetadataStorage(
       this.entity.metadata.uid,
     );
-    const { type, target } = getLocationForEntity(this.entity);
```

Comment:
we still need to check type here I think and make an exception for location types: "dir" and "file". So if it's not dir or file we should check the last commit, storage timestamp and compare them.

Reply:
Do I need to check type for github/gitlab or something else?

Reply:
you can check if type is github/gitlab and then run it in there but then we need to update the if statement every time there is a new location type added. So I think we instead just want to make a check so it's not file and dir, something like

```
if(type !== 'dir' || type !== 'file){
   .... check last commit, storage timestamp, compare etc...
}
```

Reply:
You could also do something like this to have a list of types to exclude
```javascript
const nonAgeCheckTypes = ['dir', 'file'];
if (!nonAgeCheckTypes.includes(type)) {
  // Check if docs is outdated
}
```